CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+
+## Welcome to the MyLibreLab project!
+
+First off, thank you for considering contributing to MyLibreLab. It's people like you that make open source possible! :tada:
+
+MyLibreLab is an open source project and we love to receive contributions from our community — you! There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code which can be incorporated into MyLibreLab itself.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+# Ground Rules
+
+Responsibilities
+* Ensure that code that goes into core meets all requirements in this checklist: {{CHECKLIST}}
+* Create issues for any major changes and enhancements that you wish to make. Discuss things transparently and get community feedback.
+* Be welcoming to newcomers and encourage diverse new contributors from all backgrounds.
+
+
+# Your First Contribution
+
+Unsure where to begin contributing to MyLibreLab? You can start by looking through these labels!
+* good first issue - issues which should only require a few lines of code, and a test or two.
```

Comment:
This could be [goo first issue](https://github.com/MyLibreLab/MyLibreLab/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) and [help wanted](https://github.com/MyLibreLab/MyLibreLab/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)